keccak256/src/gates/iota_b13.rs

```diff
@@ -115,7 +115,7 @@ mod tests {
             }
 
             fn configure(meta: &mut ConstraintSystem<F>) -> Self::Config {
-                let q_enable = meta.selector();
+                let q_enable = meta.complex_selector();
```

Comment:
Why have these selectors been forced to use `complex_selector` now?

Reply:
`zcash` decided not to selector in lookup.
https://github.com/zcash/halo2/commit/145d096dcd6f840178e97ec2bfd6ca9c6201093e#diff-c7d3f048cff35b9298e74eb82f24b7755a72927c699f387552c0ac97f7c6c4a8R1224